dev/docker_data/bids_utils.py

```diff
@@ -452,13 +457,17 @@ def bids_gen_cpac_sublist(bids_dir, paths_list, config_dict, creds_path, dbg=Fal
                     if "fMRI" in f_dict["acq"]:
                         if "fmap" not in subdict[f_dict["sub"]][f_dict["ses"]]:
                             subdict[f_dict["sub"]][f_dict["ses"]]["fmap"] = {}
-                        if "epi_{0}".format(pe_dir) not in subdict[f_dict["sub"]][f_dict["ses"]]["fmap"].keys():
+                        if "epi_{0}".format(
```

Comment:
Similar to a previous comment: Just ask if key in the dict as a whole, without calling the `keys()` method.

Reply:
Resolved in 72066d8